src/otk/command.py

```diff
@@ -42,12 +42,16 @@ def _process(arguments: argparse.Namespace, dry_run: bool) -> int:
 
     if arguments.input is None:
         path = pathlib.Path(f"/proc/self/fd/{sys.stdin.fileno()}")
+        # non-existing fake file to get the paths right
+        cwd = pathlib.Path.cwd() / "stdin"
```

Comment:
(quick-drive-by): maybe `<stdin>`or something here so that it looks less like a real filename (especially in errors that print the path of the fragment)?

Reply:
I'm just afraid of '<' in a filename - but yes, it's not really used :-D

Reply:
Heh, yeah, I think it's kinda-the-point to make it scary so that it's clear that it's not a real file if it leaks in e.g. an error message (which I think is quite likely, like `something wong in file "/home/mvogt/otk/stdin` vs `something wong in file "/home/mvogt/otk/<stdin>` (or maybe we can tweak the errors to something like `error in file <stdin> (dir /home/mvogt/otk): foo` i.e. splitting the path then the error woudl be nice for both case :)